### PR TITLE
api based trigger route list

### PR DESCRIPTION
**Pull request by jetjinser**

### CODE REVIEW

File: slack/src/main.rs [hunk 1]
```diff
@@ -41,6 +41,14 @@ lazy_static! {
         .timeout(Duration::from_secs(TIMEOUT))
         .build()
         .expect("Can't build the reqwest client");
+    static ref EVENTS: Value = {
```
Comment (ho-229) [inline slack/src/main.rs]: It's a good idea, but the Json content can be passed directly to the client.

Comment (jetjinser) [reply slack/src/main.rs]: I'll explain, there is a problem:

if pass json content directly as Json() params, the response just a raw string. any suggestions to solve it without serde?

Comment (ho-229) [reply slack/src/main.rs]: In fact, you don't need to go through Json(), return Json string directly and don't forget to modify the Content-Type header.

Comment (jetjinser) [reply slack/src/main.rs]: you are right, please review new commit

**Pull request merged**